<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Central_SelectedCar</class>
 <widget class="QDialog" name="Central_SelectedCar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View car</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/res/data/images/Loko.png</normaloff>:/res/data/images/Loko.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame
{
background:#111;
}

QPushButton
{
        font-size:15px;
        background:transparent;
        color:#ffffff;
}

QPushButton#HomeButton:hover, QPushButton#RentCar:hover,  QPushButton#MyRentals:hover, QPushButton#MyAccount:hover,QPushButton#signout:hover
{
        font-size:15px;
        background:#222;
        color:#ffffff;
        border-radius:50px;
}

QToolButton
{

}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>15</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#5F0000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RentCar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rent a Car</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MyRentals">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>My Rentals</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MyAccount">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>My Account</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="signout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>sign out</string>
    </property>
   </widget>
   <zorder>RentCar</zorder>
   <zorder>MyRentals</zorder>
   <zorder>HomeButton</zorder>
   <zorder>label_3</zorder>
   <zorder>MyAccount</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label</zorder>
   <zorder>signout</zorder>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>821</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#CarName, QLabel#CarNmae2, QLabel#CarBrand,  QLabel#CarBrand_2, QLabel#CarPrice, QLabel#CarPrice_2, QLabel#CarDesript
{
        color:white;
        font-size:11px;
        background-color:#333;
        border-radius:11px;
}

QTextBrowser#textBrowserDes
{
        background-color:#333;
        font-size:12px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>841</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#5F0000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
        border-image:url(L:/UNI/Projects/Project one/Grind/Project/QT/data/icons/cancel.png);

}

QPushButton:hover
{
        background:#5F0000;
        border-radius:15px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-47</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CarIm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#333;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CarName">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Model :</string>
    </property>
   </widget>
   <widget class="QLabel" name="CarNmae2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="CarBrand">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Brand :</string>
    </property>
   </widget>
   <widget class="QLabel" name="CarBrand_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="CarPrice">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Price (/hr):</string>
    </property>
   </widget>
   <widget class="QLabel" name="CarPrice_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserDes">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>581</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CarDesript">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Description :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>550</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RENT CAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
